source/_integrations/mqtt.markdown

```diff
@@ -67,7 +67,7 @@ There are [at least two](https://issues.apache.org/jira/browse/AMQ-6360) [issues
 
 The Mosquitto project runs a [public broker](https://test.mosquitto.org). This is the easiest to set up, but there is no privacy as all messages are public. Use this only for testing purposes and not for real tracking of your devices or controlling your home. To use the public mosquitto broker, configure the MQTT integration to connect to broker `test.mosquitto.org` on port 1883 or 8883.
 
-## Broker configuration
+## Configuration of the connection to the broker
```

Comment:
I rather prefer not to update the headers if this is not strictly needed as with it also the page anchors change.
To specific this header it seems clear enough to me in it current form.